### ISSUE TITLE
Lack of support for phpstan 0.11.2

### ISSUE THREAD

**Issue by IvoPereira** [proget-hq/phpstan-yii2#14]
Is there any planned upgrade for phpstan 0.11.2?

At the moment I am not able to install the extension based on the lack of support:

````
  Problem 1
    - Installation request for proget-hq/phpstan-yii2 ^0.4.2 -> satisfiable by proget-hq/phpstan-yii2[0.4.2].
    - proget-hq/phpstan-yii2 0.4.2 requires phpstan/phpstan ^0.10.3 -> satisfiable by phpstan/phpstan[0.10.3, 0.10.4, 0.10.5, 0.10.6, 0.10.7, 0.10.8, 0.10.x-dev] but these conflict with your requirements or minimum-stability.
````

Thanks

**Comment by akondas** [comment]
Yes, support for 0.11.2 is on the way :)

**Comment by IvoPereira** [comment]
Seems great @akondas. Any planned dates?

Let me know if I might help with a PR.

**Comment by marmichalski** [comment]
There is a PR already: https://github.com/proget-hq/phpstan-yii2/pull/13, feel free to contribute

**Comment by akondas** [comment]
`0.5.0` released with support for phpstan `0.11`
https://github.com/proget-hq/phpstan-yii2/releases/tag/0.5.0

**Comment by akondas** [comment]
Thanks to @marmichalski :medal_military: :medal_sports:
